### PR TITLE
add axes arg to plot_psds_topomap

### PR DESCRIPTION
**Pull request by mmagnuski**
If you want I can make this PR bigger and add axes argument in more places. 
Also - take a look at the test I've added - it takes about 40 seconds, so I may be doing something wrong...

### CODE REVIEW

**Comment by agramfort** [comment]
LGTM but keep test time really low

**Comment by mmagnuski** [comment]
takes 6 seconds now

**Comment by dengemann** [comment]
> takes 6 seconds now

and before?

Otherwise looks good to me.

**Comment by mmagnuski** [comment]
40 s -> 6 s
But I'm only talking about the part I added. test_topomap is still not-so-fast.
BTW - maybe plot_psds_topomap should have something like topo_args for more control (topomap res for example)?

**Comment by dengemann** [comment]
The improvement is great, but 6 seconds is still a lot.

>maybe plot_psds_topomap should have something like topo_args for more
control (topomap res for example)?

I am a bit reluctant  to add too much API to our plotters (but not against
it), we return matplotlib figure handles and all can be customized
post-hoc. In some cases it is very handy to have an extra API, especiall
for some of the more complicated functions that do many matplotlib API
calls internally. Keep this in mind when thinking about extending and
adding function keywords. They also incur testing costs at a quadratic
scale ...

**Comment by mmagnuski** [comment]
Yeah, 6 s is a lot, any ideas on how to speed up?
I could speed it up if I could ask plot_topomap to use lower resolution of the maps - that's why I asked about topo_args. :) I understand API concerns.

**Comment by dengemann** [comment]
@mmagnuski did you try the size parameter? It implicitly handles resolution

File: mne/viz/tests/test_topomap.py [hunk 1]
```diff
@@ -300,5 +302,16 @@ def test_plot_tfr_topomap():
...
     plt.close('all')
 
+    # test plot_psds_topomap
+    info = raw.info.copy()
+    chan_inds = channel_indices_by_type(info)
+    info = pick_info(info, chan_inds['grad'][:5])
+
+    fig, axes = plt.subplots(nrows=2)
+    freqs = np.arange(0., 13.)
+    bands = [(4, 8, 'Theta'), (8, 12, 'Alpha')]
```
Comment (dengemann) [inline mne/viz/tests/test_topomap.py]: do we need two for testing this? "#speedconcerns"

Comment (mmagnuski) [inline mne/viz/tests/test_topomap.py]: Good hashtag :) I'll change to one band today.

Comment (dengemann) [inline mne/viz/tests/test_topomap.py]: > info = pick_info(info, chan_inds['grad'][:5])

@mmagnuski you could empirically find the number of channels at which the topomap call breaks. Maybe 2 would work and save more time.

**Comment by mmagnuski** [comment]
@dengemann 
I could find `size` parameter only in

**Comment by mmagnuski** [comment]
@dengemann
I could find `size` parameter only in `plot_projs_topomap` but it doesn't control resolution (`res` does).

**Comment by agramfort** [comment]
LGTM

**Comment by jaeilepp** [comment]
Could you also add the param to ``epochs.plot_psd_topomap`` and ``plot_epochs_psd_topomap``. Like this it's a bit hard to use.

**Comment by dengemann** [comment]
Thank you @mmagnuski for efforts + patience!

**Comment by agramfort** [comment]
@mmagnuski can you address @jaeilepp comment in a next PR?

```Could you also add the param to epochs.plot_psd_topomap and plot_epochs_psd_topomap. Like this it's a bit hard to use.```

**Comment by mmagnuski** [comment]
Thanks all! I will adress @jaeilepp comments in the following PR.

**Pull request merged**